cmake/modules/BuildRocksDB.cmake

```diff
@@ -18,6 +18,7 @@ function(do_build_rocksdb)
   list(APPEND ROCKSDB_CMAKE_ARGS -DCMAKE_AR=${CMAKE_AR})
   list(APPEND ROCKSDB_CMAKE_ARGS -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE})
   list(APPEND ROCKSDB_CMAKE_ARGS -DFAIL_ON_WARNINGS=OFF)
+  list(APPEND ROCKSDB_CMAKE_ARGS -DROCKSDB_DISABLE_TBB=ON)
```

Comment:
this does not work with the cmake building system which we are using now. i just posted https://github.com/facebook/rocksdb/pull/3750 to add an option to disable the TBB support. we can update our end after the upstream gets this change merged.